src/Symfony/Component/Validator/Constraints/Unique.php

```diff
@@ -37,13 +39,25 @@ class Unique extends Constraint
     public $message = 'This collection should contain only unique elements.';
     public $normalizer;
 
+    /**
+     * {@inheritdoc}
+     *
+     * @param array|string $fields the combination of fields that must contain unique values or a set of options
+     */
     public function __construct(
         array $options = null,
         string $message = null,
         callable $normalizer = null,
         array $groups = null,
-        mixed $payload = null
+        mixed $payload = null,
+        array|string $fields = null,
     ) {
+        if (\is_array($fields) && \is_string(key($fields))) {
```

Comment:
do we need this logic now? I don't see it's purpose now that the argument is last

Reply:
> do we need this logic now? I don't see it's purpose now that the argument is last

Yes, now it makes no sense. Removed fields argument from options.